### DISCUSSION TITLE
There are so many logs, how can I disable it ?

### DISCUSSION THREAD

**Discussion by rain2307** [emilk/egui#3188] [Q&A] [answered]
I 'm using [tokio tracing](https://github.com/tokio-rs/tracing) with egui/eframe. when I run the app there are many logs like this:

```

  2023-07-29T04:38:18.435543Z TRACE eframe::native::run: Repaint caused by winit::Event: RedrawEventsCleared
    at C:\Users\xxx\.cargo\registry\src\index.crates.io-6f17d22bba15001f\eframe-0.22.0\src\native\run.rs:205

  2023-07-29T04:38:18.435791Z TRACE eframe::native::run: request_redraw
    at C:\Users\xxx\.cargo\registry\src\index.crates.io-6f17d22bba15001f\eframe-0.22.0\src\native\run.rs:221

  2023-07-29T04:38:18.436249Z TRACE eframe::native::run: Got outdated UserEvent::RequestRepaint
    at C:\Users\xxx\.cargo\registry\src\index.crates.io-6f17d22bba15001f\eframe-0.22.0\src\native\run.rs:160
```

How can I disable it ?

**Comment by rain2307** [comment]
any body?help?

**Comment by emilk** [comment, marked as answer]
You have configured you logger at the verbose TRACE level.
Raise the log level with `RUST_LOG=debug`, or specify a higher log level for eframe with `RUST_LOG=trace,eframe=info`.

**Reply by rain2307** [reply]
Thank You! But `RUST_LOG=debug` is not working...
I have tried:

```bash
RUST_LOG=debug cargo run
RUST_LOG=trace,eframe=info cargo run 
```
Both not working.

I'm using gitbash on windows.

**Reply by rain2307** [reply]
[I create a gist to show how I init tracing](https://gist.github.com/rain2307/82c0129196ed76f0f1b28d1a5534d1a4)

**Comment by rain2307** [comment]
I should ask this in tracing repo. Thanks.
